### PR TITLE
Feature/47 configure logging

### PR DESCRIPTION
**Pull request by OksanaOlesja**

### CODE REVIEW

**Review by AndriyBarskyi — APPROVED** [review]

File: src/main/java/com/softserve/edu/sporthubujp/controller/ArticleController.java [hunk 1]
```diff
@@ -12,6 +16,7 @@
 @RequestMapping("api/v1/articles")
 
 public class ArticleController {
+    private static final Logger LOG = Logger.getLogger(ArticleController.class.getName());
```
Comment (yuremboo) [inline src/main/java/com/softserve/edu/sporthubujp/controller/ArticleController.java:19]: you can use @ Slf4j annotation instead this line

File: src/main/java/com/softserve/edu/sporthubujp/controller/ArticleController.java [hunk 2]
```diff
@@ -21,12 +26,13 @@ public ArticleController(ArticleService articleService) {
 
     @GetMapping("/{id}")
     public ResponseEntity<ArticleDTO> getArticleById(@PathVariable String id) {
+        LOG.log(Level.INFO, "Get article by id {0}", id);
```
Comment (yuremboo) [inline src/main/java/com/softserve/edu/sporthubujp/controller/ArticleController.java:29]: LOG.info("Get article by id {}", id);

File: src/main/java/com/softserve/edu/sporthubujp/controller/ArticleController.java [hunk 3]
```diff
@@ -21,12 +26,13 @@ public ArticleController(ArticleService articleService) {
...
         return ResponseEntity.status(HttpStatus.OK).body(
                 articleService.getArticleById(id));
     }
-    @DeleteMapping(path = "/{id}")
-    public ResponseEntity<Void> deleteArticle(@PathVariable("id") String articleId)
-    {
+    @DeleteMapping("/{id}")
+    public ResponseEntity<Void> deleteArticle(@PathVariable("id") String articleId) {
+        LOG.log(Level.INFO, "Delete article by id {0}", articleId);
```
Comment (yuremboo) [inline src/main/java/com/softserve/edu/sporthubujp/controller/ArticleController.java:35]: the same

File: src/main/java/com/softserve/edu/sporthubujp/service/impl/ArticleServiceImpl.java [hunk 4]
```diff
@@ -31,8 +35,11 @@ public ArticleDTO getArticleById(String id) {
     @Override
     public void deleteArticleById(String id)
     {
+        LOG.info("Delete article by id in service");
         if(!articleRepository.existsById(id))
         {
+            //LOG.log(Level.WARNING,"Record with provided id is not found" );
+            LOG.throwing(ArticleServiceImpl.class.getName(),"deleteArticleById",new ArticleServiceException("Record with provided id is not found"));
```
Comment (yuremboo) [inline src/main/java/com/softserve/edu/sporthubujp/service/impl/ArticleServiceImpl.java:42]: LOG.error(...)

**Review by yuremboo — CHANGES_REQUESTED** [review]

**Review by NastiaChaikovska — APPROVED** [review]

**Review by yuremboo — APPROVED** [review]

**Pull request merged**